### PR TITLE
[TM-1536] add total_content and proccessed_content field to delayedJo…

### PR DESCRIPTION
**Pull request by LimberHope**
[Task Link](https://gfw.atlassian.net/browse/TM-1536)

### CODE REVIEW

**Review by pachonjcl — APPROVED** [review]

File: apps/job-service/src/jobs/dto/delayed-job.dto.ts [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 
 const STATUSES = ["pending", "failed", "succeeded"];
 type Status = (typeof STATUSES)[number];
 
 @JsonApiDto({ type: "delayedJobs" })
 export class DelayedJobDto extends JsonApiAttributes<DelayedJobDto> {
+  constructor(job: DelayedJob) {
+    const { status, statusCode, payload, processed_content, total_content, proccess_message } = job;
+    super({ status, statusCode, payload, processed_content, total_content, proccess_message });
+  }
```
Comment (roguenet) [inline apps/job-service/src/jobs/dto/delayed-job.dto.ts:14]: Why was this added? It should not be necessary.
